### DISCUSSION TITLE
v2 terminal - How to hide left frame for a single project?

### DISCUSSION THREAD

**Discussion by depsimon** [vercel/turborepo#8352] [Help]
### Summary

In v1 I was setting up tasks in my root package.json that ran `turbo run task --filter=app --log-prefix=none`.

It ran the task without any prefix so it was just like a normal task.

I'd like to be able to do the same with the v2 terminal.

### Additional information

```sh
turbo run dev --filter=@acme/app --log-prefix=none
```

It should not render a left column as we don't want a prefix and we selected a specific project.

**Comment by anthonyshew** [comment]
I'm thinking this makes sense as the default behavior. I'll bring this up with the team to see if we can get this change.

**Reply by anthonyshew** [reply]
Going to close for duplicate of #8351
